HUD/BeginTextCommandDisplayText.md

```diff
@@ -9,14 +9,31 @@ aliases: ["_SET_TEXT_ENTRY"]
 void BEGIN_TEXT_COMMAND_DISPLAY_TEXT(char* text);
 ```
 
-```
-The following were found in the decompiled script files:  
-STRING, TWOSTRINGS, NUMBER, PERCENTAGE, FO_TWO_NUM, ESMINDOLLA, ESDOLLA, MTPHPER_XPNO, AHD_DIST, CMOD_STAT_0, CMOD_STAT_1, CMOD_STAT_2, CMOD_STAT_3, DFLT_MNU_OPT, F3A_TRAFDEST, ES_HELP_SOC3  
-ESDOLLA   
-ESMINDOLLA - cash (negative)  
-Used to be known as _SET_TEXT_ENTRY  
```

Comment:
Also, these are useful, not sure why you would get rid of them.

Reply:
My reasoning behind this was that these entries are just some random entries from the global.gxt2 files, which to me at the time seemed more appropriate to be either in "GetLabelText" or "AddLabelText" natives since those are specific to the labels.

I could also refer to GetLabelText and move the example labels over to there, or should I just add it back to the description it self?

Reply:
I've changed it around a bit, let me know if this is preferrable, or if the example labels would still be usefull to add in BegintTextCommandDisplayText

Reply:
> I've changed it around a bit, let me know if this is preferrable, or if the example labels would still be usefull to add in BegintTextCommandDisplayText

This is way better!

There's still a typo though:
`BegintTextCommandDisplayText` should be `BeginTextCommandDisplayText`.

If you could correct that, it would be great, but so far this is going in the right direction from what I can see 😊

Reply:
Ah yes, I missed that part, fixed now